src/sentry/api/endpoints/prompts_activity.py

```diff
@@ -79,13 +79,21 @@ def put(self, request):
         if any(elem is None for elem in fields.values()):
             return Response({'detail': 'Missing required field'}, status=400)
 
+        # if project_id or organization_id in required fields make sure they exist
+        # if NOT in required fields, insert dummy value so dups aren't recorded
         if 'project_id' in required_fields:
-            try:
-                Project.objects.get(
-                    id=fields['project_id'],
-                )
-            except Project.DoesNotExist:
+            project = Project.objects.filter(id=fields['project_id']).exists()
```

Comment:
nit: I'd just condense this to:

```python
if not Project.objects.filter(id=fields['project_id']).exists():
```

It's also shorter than assigning it to a variable, which, arguable is misleading, since it's not a project, it's a boolean.